test/trento/domain/host/host_test.exs

```diff
@@ -651,6 +651,48 @@ defmodule Trento.HostTest do
   end
 
   describe "deregistration" do
+    test "should reject all the commands except the registration ones when the host is deregistered" do
+      host_id = Faker.UUID.v4()
+      dat = DateTime.utc_now()
```

Comment:
You don't really need a variable for this (`dat`), you could run this in the event itself, as it is not used anywhere else